invenio_rdm_pure/cli.py

```diff
@@ -1,3 +1,124 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# Invenio-Records-Marc21 is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CLI commands for Invenio-RDM-Pure."""
+
+
+import json
+import random
+from datetime import date, timedelta
+from os.path import dirname, isfile, join
+
+import click
+from faker import Faker
+from flask.cli import with_appcontext
+from flask_principal import Identity
+from invenio_access.permissions import any_user
+from invenio_records_marc21.services import Marc21RecordService, Metadata
+from invenio_records_marc21.vocabularies import Vocabularies
+
+from .source.rdm.converter import Converter, Marc21Record
+from .source.rdm.database import RdmDatabase
+from .source.utils import load_file_as_string
+
+
+def fake_access_right():
+    """Generate fake access_right."""
+    vocabulary = Vocabularies.get_vocabulary("access_right")
+    _type = random.choice(list(vocabulary.data.keys()))
+    return _type
+
+
+def fake_feature_date(days=365):
+    """Generate fake feature_date."""
+    start_date = date.today()
+    random_number_of_days = random.randrange(days)
+    _date = start_date + timedelta(days=random_number_of_days)
+    return _date.strftime("%Y-%m-%d")
+
+
+def create_invenio_record(record: dict) -> None:
+    """Create a record."""
+    converter = Converter()
+    record_marc21 = converter.convert_pure_json_to_marc21_xml(record)
+    if Marc21Record.is_valid_marc21_xml_string(record_marc21):
```

Comment:
Suggestion: My personal preference is to a empty line before of the if statement. Coding blocks improve the readability. Also a personal preference is to return early if a error occur. So in the sense of if not Marc21Record.is_valid_marc21_xml_string: click.secho("...") return. The improvement for the readability is that the if block does not have to be indented.

Reply:
I see what you mean, the changes have been implemented.